lib/open_project/backlogs/engine.rb

```diff
@@ -132,10 +132,57 @@ def self.settings
       :ProjectsController, :ProjectsHelper, :Query, :User, :VersionsController, :Version]
 
     extend_api_response(:v3, :work_packages, :work_package) do
-      property :story_points, if: -> (*) { backlogs_enabled? }
-      property :remaining_hours, if: -> (*) { backlogs_enabled? }
+      property :story_points,
+               render_nil: true,
+               if: -> (*) { backlogs_enabled? }
+
+      property :remaining_time,
+               exec_context: :decorator,
+               getter: -> (*) {
+                 datetime_formatter.format_duration_from_hours(represented.remaining_hours,
+                                                               allow_nil: true)
+               },
+               render_nil: true,
+               if: -> (*) { represented.backlogs_enabled? }
     end
 
+    extend_api_response(:v3, :work_packages, :form, :work_package_payload) do
+      property :story_points,
+               render_nil: true,
+               if: -> (*) { backlogs_enabled? }
+
+      property :remaining_time,
+               exec_context: :decorator,
+               getter: -> (*) {
+                 datetime_formatter.format_duration_from_hours(represented.remaining_hours,
+                                                               allow_nil: true)
+               },
+               setter: -> (value, *) {
+                 remaining = datetime_formatter.parse_duration_to_hours(value,
+                                                                        'remainingTime',
+                                                                        allow_nil: true)
+                 represented.remaining_hours =  remaining
+               },
+               render_nil: true,
+               if: -> (*) { represented.backlogs_enabled? }
+    end
+
+    extend_api_response(:v3, :work_packages, :schema, :work_package_schema) do
+      schema :story_points,
+             type: 'Integer',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.